code/infrastructure-api/manifests/jupyter.deployment.template.yml

```diff
@@ -14,6 +13,23 @@ spec:
     spec:
       securityContext:
         runAsUser: 0
+      initContainers:
```

Comment:
Worth a comment in here to explain this as it could be confusing if you don't know why these are here.

Reply:
Very clever though!